Pipfile

```diff
@@ -4,10 +4,10 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-geojson = ">=1.3.1"
-pytest = "6.0.0"
-requests = ">=2.20.0"
-shapely = ">=1.6.4"
+geojson = ">=3.0.1"
+pytest = "7.4.0"
+requests = ">=2.31.0"
+shapely = ">=2.0.1"
```

Comment:
New major version of shapely, are we confident we won't break things on our end?